public/js/main.js

```diff
@@ -92,12 +93,17 @@ if (connTarget) {
   });
 } else if (isFirefoxPanel()) {
   // The toolbox already provides the tab to debug.
+  function bootstrap({ threadClient, tabTarget }, callback) {
+    firefox.setThreadClient(threadClient);
+    firefox.setTabTarget(tabTarget);
+    firefox.initPage(actions);
+    renderRoot(App);
+
+    return { store, actions, selectors };
```

Comment:
i changed this version slightly to pass `store, actions, selectors` instead of `getState` and `selectors`